SR2019/2019-04-23-competition-wrap-up.md

```diff
@@ -0,0 +1,52 @@
+---
+to: Student Robotics 2019
+subject: Final competition details
+---
+
+![Competition Photo](https://studentrobotics.org/images/content/news/sr2019/sr2019-photo.jpg)
+
+Hi all,
+
+First off, we'd like to thank you for taking part in Student Robotics 2019 and making it such a successful event! An article about the competition, results, and game has been [posted on our website](https://studentrobotics.org/news/2019-04-09-sr2019-tlc-crowned-champion/)
+
+Congratulation again to our winners:
+
+|        Prize          |            Team
+|-----------------------|-----------------------------------------------
+| 1st Place             | [The Ladies' College Guernsey](http://www.ladiescollege.com/)
+| 2nd Place             | [Hills Road Sixth Form College](http://www.hillsroad.ac.uk/)
+| 3rd Place             | [Hampton School](https://hamptonschool.org.uk/) and [Lady Eleanor Holles School](https://www.lehs.org.uk/)
+| Rookie Award          | [Eltham College](https://www.elthamcollege.london/)
+| Committee Award       | [South Wilts Grammar School](https://www.swgs.wilts.sch.uk/)
+| Robot and Team Image  | [Haberdashers' Aske's Boys' School](https://www.habsboys.org.uk/)
+| Online Presence       | [Collyer's](https://www.collyers.ac.uk/)
+
+## Event Photos
+
+We've collected together all the media together into 1 album. If you wish to use and republish any of the photos in college material, please be sure to attribute them to [Student Robotics](https://studentrobotics.org).
```

Comment:
Maybe inline the link (and use words for small numbers in prose):
```suggestion
We've collected together all the media together into [an album](https://photos.app.goo.gl/8vtBZiuM8FXSf79Z8). If you wish to use and republish any of the photos in college material, please be sure to attribute them to [Student Robotics](https://studentrobotics.org).
```
(and then drop the "Album link" below).